### DISCUSSION TITLE
RED Square When i add or remove shorcut on homw "new-tab"

### DISCUSSION THREAD

**Discussion by validMyID** [Alex313031/Thorium-Win#246] [General]
![image](https://github.com/user-attachments/assets/e341d838-04af-4622-85d3-8ea3d4a10d30)
hello,
i found this
is this bug or only my side?

edit:
i found here too
![image](https://github.com/user-attachments/assets/1097b529-9103-4ade-afd2-c5d253729ff2)

and this one (but i think someone already report this
![image](https://github.com/user-attachments/assets/0bbb3a06-81b9-4eff-a583-97c18c0630e1)

**Comment by gz83** [comment]
Please try the latest version 138. If you still have problems, please let us know.
